model/k8s/metrics.yaml

```
@@ -469,3 +469,21 @@ groups:
     attributes:
       - ref: k8s.namespace.phase
         requirement_level: required
+
+  # k8s.container.* metrics
+  - id: metric.k8s.container.restart.count
+    type: metric
+    metric_name: k8s.container.restart.count
+    stability: development
+    brief: "Describes how many times the container has restarted in the recent past. "
```

```suggestion
    brief: "Describes how many times the container has restarted (since the last counter reset)"
```

?

the recent past is very ambiguous, let's try to be more precise

---

fixed.